scene/2d/gpu_particles_2d.cpp

```diff
@@ -34,6 +34,11 @@
 
 #ifdef TOOLS_ENABLED
 #include "core/config/engine.h"
+
+void GPUParticles2D::set_show_visibility_rect(bool p_show_visibility_rect) {
+	show_visibility_rect = p_show_visibility_rect;
+	update();
+}
```

Comment:
It would be better to place this further down, matching its position in the header, so after `set_trail_section_subdivisions`. Here it looks as if it was directly linked to the `#include` enclosed in this `#ifdef`, which is not the case.

Reply:
done!